### ISSUE TITLE
Build log should point to the comment which gave command

### ISSUE THREAD

**Issue by tannakartikey** [yegor256/rultor#1198]
In each build log, the first line states from where the build was triggered
e.g. This build log was produced by [rultor.com](http://doc.rultor.com) for tannakartikey/currents#13

The URL should point to the talk on the Issue/PR page.
As Rultor might have been given commands multiple times from the same Issue/PR.

The build log URL includes the talk number, but tracking which talk is responsible for the build log through a URL is not very convenient.

**Comment by 0crat** [comment]
@yegor256 please, pay attention to this issue

**Comment by yegor256** [comment]
@tannakartikey this totally makes sense. Would you be interested in contributing with a pull request?

**Comment by tannakartikey** [comment]
Yes, I am. But this will be my first time with Java. So might take a while.

On 18-Jul-2017 2:59 PM, "Yegor Bugayenko" <notifications@github.com> wrote:

@tannakartikey <https://github.com/tannakartikey> this totally makes sense.
Would you be interested in contributing with a pull request?

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/yegor256/rultor/issues/1198#issuecomment-316009561>,
or mute
the thread
